### PR TITLE
DOC-345 | Arangograph Data Loader

### PR DESCRIPTION
**Pull request by nerpaula**
### Description

Preview: https://deploy-preview-328--docs-hugo.netlify.app/stable/arangograph/data-loader/

TO DO:

- [x] apply to all versions
- [x] revise screenshots
- [x] add **Import** description once finalized
- [x] add more info about the parsing errors and import helper once finalized
- [x] add end-to-end example

### CODE REVIEW

**Review by ansoboleva — APPROVED** [review]
lgtm

**Comment by Simran-B** [comment]
With the tabs, it takes 3 clicks to read all 4 sentences. What purpose that this serve?

**Comment by nerpaula** [comment]
> With the tabs, it takes 3 clicks to read all 4 sentences. What purpose that this serve?

@Simran-B IMO it's not all about "how many clicks it takes", this applies mostly to actions you do in the web interface. But when it comes to docs, sometimes one does need a bit of visual mix and only scrolling may feel dull.

**Comment by Simran-B** [comment]
An ordered list can also add visual interest without making the content inaccessible to in-page search:

image

**Comment by Simran-B** [comment]
Maybe noteworthy: the data loader supports `;` as a delimiter and values with line breaks in them if wrapped in quotes.
All values are imported as strings, however.

"File headers" is a bit of an odd choice for a label as it isn't clear that this is for selecting fields to be imported. There should be an option to tick all boxes IMO (perhaps excluding the ones already used for _key, _from, and _to). The example values column could use a label to explain what this is.

File: site/content/3.10/arangograph/data-loader/import.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  Once the data files are provided and the graph is designed, you can start the import
+archetype: default
+---
+
+Before starting the actual import, make sure that:
+- You have selected a database for import or created a new one;
+- You have provided a name for your graph;
+- You have created at least one node;
+- You have created at least one edge;
+- Every file is related to at least one node or edge.
```
Comment (shd8) [inline site/content/3.10/arangograph/data-loader/import.md:15]: FYI, the full list of requirements for import is:

- You have selected a database for import or created a new one;
- You have provided a valid name for your graph;
- The graph name provided is not in use already;
- You have created at least one node;
- You have created at least one edge;
- You have uploaded at leat one file;
- Every file is related to at least one node or edge;
- Every node and edge is linked to a file;
- Every node and edge has a valid label;
- Every node and edge label is unique;
- Every node has a primary identifier selected;
- Every edge has an origin and destination file header selected;

Maybe is not needed to include all restrictions to the official list, but this is the list.

Comment (nerpaula) [reply site/content/3.10/arangograph/data-loader/import.md]: Thanks, updated the list.

**Pull request merged**